var/spack/repos/builtin/packages/openfoam-org/package.py

```diff
@@ -126,24 +137,37 @@ def config(self):
             settings['label-size'] = False
         return settings
 
-    def setup_environment(self, spack_env, run_env):
+    def setup_build_environment(self, env):
```

Comment:
I would get rid of this function

Reply:
That function was a relic from openfoam-package that has a similar unused function. Indeed copying it was a bit unnecessary.